### PR TITLE
FACES-2938 Upgrade Bridge integration tests to the new test framework (add maven profiles to test the showcases seperately)

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: test/integration/pom.xml [hunk 1]
```diff
@@ -210,22 +210,184 @@
...
+								</configuration>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+				<pluginManagement>
+					<plugins>
+						<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself. -->
+						<plugin>
+							<groupId>org.eclipse.m2e</groupId>
```
Comment (ngriffin7a) [inline test/integration/pom.xml:229]: @stiemannkj1 This is already merged and backported but is the org.eclipse.m2e section necessary? Those sections were removed from all of the project poms when the project moved from the monolithic Git repository to the separate repositories.

Comment (stiemannkj1) [reply test/integration/pom.xml:229]: @ngriffin7a, I don't think it's necessary. I just was following some conventions (maybe I copy/pasted old code).

Comment (ngriffin7a) [reply test/integration/pom.xml:229]: @stiemannkj1 I removed the m2e settings in a code review commit and backported.

Comment (stiemannkj1) [reply test/integration/pom.xml:229]: Thanks!

**Pull request merged**